main-command/src/main/scala/sbt/Command.scala

```diff
@@ -91,16 +91,16 @@ object Command {
   /** This is the main function State transfer function of the sbt command processing, called by MainLoop.next, */
   def process(exec: Exec, state: State): State =
     {
-      val channelName = exec.source map { _.channelName }
-      State.exchange.publishEventMessage(ExecStatusEvent("Processing", channelName, exec.execId, Vector()))
+      //      val channelName = exec.source map { _.channelName }
+      //      State.exchange.publishEventMessage(ExecStatusEvent("Processing", channelName, exec.execId, Vector()))
```

Comment:
We still need this, so we might have to move `Command.process` to somewhere else.